include/sysmallocheap.hpp

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#ifndef NEXTHEAP_HPP
```

Comment:
should be renamed to `SYSMALLOCHEAP_HPP`

Reply:
Oops